docs/en/04_Changelogs/4.8.0.md

```diff
@@ -10,47 +10,58 @@
 
 ### Support for silverstripe/graphql v4 {#graphql-v4}
 
-The [silverstripe/graphql](http://github.com/silverstripe/silverstripe-graphql/issues) module
-to drive various parts of the CMS UI, as well as building your own GraphQL APIs in Silverstripe projects.
+The [silverstripe/graphql](http://github.com/silverstripe/silverstripe-graphql/issues) module is used
+to drive various parts of the CMS UI, as well as building your own GraphQL APIs in your Silverstripe CMS projects.
 
-The CMS currently ships with `silverstripe/graphql:^3`, which is stable but slow for larger schemas.
-The module was rewritten from scratch with performance in mind
-(see [goals](https://github.com/silverstripe/silverstripe-framework/issues/8824) and [pull request](https://github.com/silverstripe/silverstripe-graphql/pulls)).
-The resulting`silverstripe/graphql:^4` release is currently in a alpha stage.
-We encourage everyone to get started, read our new [GraphQL documentation](https://docs.silverstripe.org/en/4/developer_guides/graphql/), 
-provide [feedback](https://github.com/silverstripe/silverstripe-graphql/issues) and help us stabilise the module.
```

Comment:
I moved this bit to a dedicated section for people who want to upgrade to v4 right now.